client_restaurant/shared/src/commonMain/kotlin/data/remote/model/RestaurantPermission.kt

```diff
@@ -0,0 +1,22 @@
+package data.remote.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class RestaurantPermission(
```

Comment:
I think we should be named `RestaurantPermissionDto` and you don't have to provide default values; just make it nullable.